### PR TITLE
Add hybrid quantization for Flux model

### PR DESCRIPTION
**Pull request by l-bat**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Comment by l-bat** [comment]
@AlexKoff88, @nikita-savelyevv please take a look

File: optimum/intel/openvino/quantization.py [hunk 1]
```diff
@@ -1150,7 +1150,14 @@ def _hybrid_quantization(
...
     wc_config.ignored_scope = wc_config.ignored_scope or {}
-    wc_config.ignored_scope["types"] = wc_config.ignored_scope.get("types", []) + ["Convolution"]
+
+    wc_ignored_scope = []
+    for op in model.get_ops():
+        if op.get_type_name() == "Convolution":
+            wc_ignored_scope.append("Convolution")
+            break
+
+    wc_config.ignored_scope["types"] = wc_config.ignored_scope.get("types", []) + wc_ignored_scope
```
Comment (nikita-savelyevv) [inline optimum/intel/openvino/quantization.py:1160]:
```suggestion
    wc_ignored_types = ["Convolution"] if any(op.get_type_name() == "Convolution" for op in model.get_ops()) else []
    wc_config.ignored_scope["types"] = wc_config.ignored_scope.get("types", []) + wc_ignored_types
```

**Review by nikita-savelyevv — APPROVED** [review]

**Review by AlexKoff88 — APPROVED** [review]

**Comment by AlexKoff88** [comment]
@echarlaix, @IlyasMoutawwakil, this is what we forgot about when enabling FLUX.

**Review by IlyasMoutawwakil — APPROVED** [review]
LGTM ! approve merge as long as tests are passing

**Pull request merged**